File: Android project/AndroidNetworkniginJava/app/src/main/java/com/mirkamol/androidnetworkniginjava/network/utils/Logger.java
package com.mirkamol.androidnetworkniginjava.network.utils;

import android.util.Log;

import com.mirkamol.androidnetworkniginjava.network.volley.VolleyHttp;

public class Logger {

   public static void d(String tag, String msg) {
        if (VolleyHttp.IS_TESTER)
            Log.d(tag,  msg);
    }

   public static void i(String tag, String msg) {
        if (VolleyHttp.IS_TESTER)
            Log.d(tag,  msg);
    }

   public static void v(String tag, String msg) {
        if (VolleyHttp.IS_TESTER)
            Log.d(tag,  msg);
    }

  public   static void e(String tag, String msg) {
        if (VolleyHttp.IS_TESTER)
            Log.d(tag,  msg);
    }


}

File: Android project/OgabekDevBestPractise/app/src/main/java/com/mirkamol/ogabekdevbestpractise/helper/Logger.kt
package com.mirkamol.ogabekdevbestpractise.helper

import android.util.Log
import com.mirkamol.ogabekdevbestpractise.networking.RetrofitHttp

object Logger {
    fun d(tag: String?, msg: String?) {
        if (RetrofitHttp.IS_TESTER) Log.d(tag, msg!!)
    }

    fun i(tag: String?, msg: String?) {
        if (RetrofitHttp.IS_TESTER) Log.i(tag, msg!!)
    }

    fun v(tag: String?, msg: String?) {
        if (RetrofitHttp.IS_TESTER) Log.v(tag, msg!!)
    }

    fun e(tag: String?, msg: String?) {
        if (RetrofitHttp.IS_TESTER) Log.e(tag, msg!!)
    }
}